docs/types.rst

```diff
@@ -50,13 +50,51 @@ Operators:
 
 * Comparisons: ``<=``, ``<``, ``==``, ``!=``, ``>=``, ``>`` (evaluate to ``bool``)
 * Bit operators: ``&``, ``|``, ``^`` (bitwise exclusive or), ``~`` (bitwise negation)
-* Arithmetic operators: ``+``, ``-``, unary ``-``, unary ``+``, ``*``, ``/``, ``%`` (remainder), ``**`` (exponentiation), ``<<`` (left shift), ``>>`` (right shift)
+* Arithmetic operators: ``+``, ``-``, unary ``-``, ``*``, ``/``, ``%`` (remainder), ``**`` (exponentiation), ``<<`` (left shift), ``>>`` (right shift)
 
-Division always truncates (it is just compiled to the ``DIV`` opcode of the EVM), but it does not truncate if both
-operators are :ref:`literals<rational_literals>` (or literal expressions).
+
+Comparisons
+~~~~~~~~~~~
+
+The value of a comparison is the one obtained by comparing the integer value.
+
+Bit operations
```

Comment:
Maybe we should mention here again that shifts are arithmetic shifts, are considered to be arithmetic operators, not bit operators, and explained below in detail? I would usually consider shifts to be bit operations and a reader might miss the fact that signed right shifts are arithmetic, if it's not mentioned here...

Reply:
I'll just move the shift operations directly after "bit operations".